features/testbot/distance_matrix.feature

```diff
@@ -124,7 +124,7 @@ Feature: Basic Distance Matrix
             | d | 20 | 30  | 0  | 30 |
             | e | 30 | 40  | 10 | 0  |
 
-    Scenario: Testbot - Travel time matrix and with only one source
+    Scenario: Testbot - Rectangular travel time matrix
```

Comment:
should we ensure these are tested over partition-boundaries? I can't see any partition parameters specified here. Since we do dedicated search, it could be helpful to actually go across boundaries/test pruning to not keep values from us